spec/features/featured_item_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+RSpec.describe "featuring items" do
+  context "when viewing a featured item" do
+    let(:user) do
+      u = FactoryGirl.build(:user)
+      u.group_list = "admin"
+      u.save
+      u
+    end
+    it "should have a working unfeature link", :js => true do
```

Comment:
I think we should leave it because it communicates that the test depends on js. As another issue, I dont think we should always use the js driver, it's slower.